instrumentserver/config.py

```diff
@@ -63,6 +64,12 @@ def loadConfig(configPath: str):
         else:
             guiConfig[instrumentName] = GUIFIELD
 
+        if 'pollingRate' in configDict:
+            pollingRateList = configDict.pop('pollingRate')
+            if pollingRateList != None:
```

Comment:
change the != for `is not`. When dealing with `None` that is a better way of doing that comparison in python